lib/search_solr_tools/helpers/iso_namespaces.rb

```diff
@@ -7,25 +9,25 @@ def self.namespaces(doc = nil)
       end
 
       ISO_NAMESPACES = {
-        'atom'   => 'http://www.w3.org/2005/Atom',
```

Comment:
This was done by the rubocop "autocorrect".  I can probably move it back and update a rubocop setting to use a value-aligned style.